instrumentation/aws-sdk/aws-sdk-1.11/library/src/main/java/io/opentelemetry/instrumentation/awssdk/v1_11/AwsSdkTracing.java

```diff
@@ -37,6 +40,16 @@ public static AwsSdkTracing create(OpenTelemetry openTelemetry) {
     return builder(openTelemetry).build();
   }
 
+  /**
+   * Returns a new {@link AwsSdkTracing} configured with the given {@link OpenTelemetry} and from
+   * the global agent {@link Config}.
+   */
+  public static AwsSdkTracing fromAgentConfig(OpenTelemetry openTelemetry) {
+    return builder(openTelemetry)
+        .setCaptureExperimentalSpanAttributes(CAPTURE_EXPERIMENTAL_SPAN_ATTRIBUTES_DEFAULT)
+        .build();
+  }
```

Comment:
I don't think we want to expose "agent" methods in library instrumentation, can the agent call `setCaptureExperimentalSpanAttributes` directly instead?

Reply:
@trask I inlined the function on the two places where it is used.
This currently means that also `Config.get().getBoolean("otel.instrumentation.aws-sdk.experimental-span-attributes", false))` is duplicated.
Let me know if you prefer this to be put in a constant on the `AwsSdkTracing` class or its builder.

Reply:
thx! we have some related work to do (and will likely end up where you were going originally with adding Config default directly to library instrumentation) but appreciate keeping it consistent with existing instrumentation for now 👍